apps/studio/src/server/context.ts

```diff
@@ -41,10 +42,18 @@ export const createContext = async (opts: CreateNextContextOptions) => {
     session,
   })
 
+  const growthbookContext = new GrowthBook({
```

Comment:
The linked doc is for FE, here we are using it on the server side. Docs [here](https://docs.growthbook.io/lib/node)

> GrowthBook instances are scoped to a single incoming request. The easiest way to do this is with Middleware

Since each request gets its own context, we can add it to the context's scope right? Lmk if am missing something

Reply:
mmm yup. activesg has a specific util to use there